### PR TITLE
Added info to troubleshooting

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description

Added information to the SQL verified source troubleshooting section regarding issues with connecting to MS SQL Server using ConnectorX.

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md [hunk 1]
```diff
@@ -559,11 +559,19 @@ sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_dat
...
 ```
 
-***To use long strings (>8k) and avoid collation errors**:
+**To use long strings (>8k) and avoid collation errors**:
 ```toml
 sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_data?LongAsMax=yes&driver=ODBC+Driver+17+for+SQL+Server"
 ```
 
+**To fix MS SQL Server connection issues with ConnectorX**:
+
```
Comment (adrianbr) [inline docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md:568]: add "Some users have reported issues with MS SQL server and connector x. The issues are outside of dlt and seem to be around how  aconnection is created. Here's a user suggested solution (Thanks @User)

Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md:568]: @markjames

Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md:568]: @markjamesm

Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md:568]: @dat-a-man will you add this?
"Some users have reported issues with MS SQL Server and Connector X. The problems are not caused by dlt, but by how connections are made. A big thanks to [Mark-James M](https://github.com/markjamesm) for suggesting a solution."

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]
@dat-a-man will you add this?
"Some users have reported issues with MS SQL Server and Connector X. The problems are not caused by dlt, but by how connections are made. A big thanks to [Mark-James M](https://github.com/markjamesm) for suggesting a solution."

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md (hunk 1 above)
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md:570]: updated

**Comment by burnash** [comment]
@dat-a-man please resolve conflicts and prepare for merging if this content is still relevant.

**Review by AstrakhantsevaAA — APPROVED** [review]

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database/troubleshooting.md [hunk 2]
```diff
@@ -56,6 +56,16 @@ sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_dat
...
 
+**To fix MS SQL Server connection issues with ConnectorX**:
+
+Some users have reported issues with MS SQL Server and Connector X. The problems are not caused by dlt, but by how connections are made. A big thanks to [Mark-James M](https://github.com/markjamesm) for suggesting a solution.
+
+To fix connection issues with ConnectorX and MS SQL Server, include both `Encrypt=yes` and `encrypt=true` in your connection string:
+```py
+sources.sql_database.credentials="mssql://user:password@server:1433/database?driver=ODBC+Driver+17+for+SQL+Server&Encrypt=yes&encrypt=true"
+```
+This approach can help resolve connection-related issues.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/sql_database/troubleshooting.md:67]:
```suggestion
This approach can help resolve connection-related issues. 
```

**Review by AstrakhantsevaAA — COMMENTED** [review]
just to rerun tests

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database/troubleshooting.md [hunk 3]
```diff
@@ -56,6 +56,16 @@ sources.sql_database.credentials="mssql+pyodbc://loader:loader@localhost/dlt_dat
...
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/sql_database/troubleshooting.md:67]:
```suggestion
This approach can help resolve connection-related issues.
```

**Review by AstrakhantsevaAA — COMMENTED** [review]
rerun tests

**Review by AstrakhantsevaAA — APPROVED** [review]

**Pull request merged**